samples/nrf9160/secure_services/Kconfig

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+source "Kconfig.zephyr"
+
+menu "Secure Service Sample"
+
+config SAMPLE_SERVICE_RNG
+	bool "Enable code that the spm_request_random_number()"
```

Comment:
words

Reply:
I accidentally the spm_request_random_number() :P

Fixed